cli.py

```
@@ -4,24 +4,110 @@
+        """Load and validate pending events from feedback_pending.json."""
+        path = Path("feedback_pending.json")
+        if not path.exists():
+            return []
+
+        try:
+            with open(path) as f:
+                raw = json.load(f)
+            if not isinstance(raw, list):
+                return []
+            return [FeedbackEvent.model_validate(e) for e in raw]
+        except (json.JSONDecodeError, Exception):
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)